src/rpm/triggers.systemd.in

```diff
@@ -56,33 +44,31 @@ if posix.access("%{_localstatedir}/lib/rpm-state/systemd/needs-reload") then
     end
 end
 
-%transfiletriggerin -P 100700 -p <lua> -- @sysusersdir@
--- This script will process files installed in @sysusersdir@ to create
--- specified users automatically. The priority is set such that it
--- will run before the tmpfiles file trigger.
+%transfiletriggerpostun -P 10000 -p <lua> -- @systemunitdir@ /etc/systemd/system
```

Comment:
This should run _after_ all other systemd triggers, since we may need tmpfiles and users populated.

Reply:
Hmm, is there some place where **all** the different trigger types are documented? I.e. how does %post, %postun, %triggerin, %transfiletriggerin, %triggerun, %transfiletriggerpostun, %postrans, %pretrans, and with different priorities, get sorted?

Reply:
🤷🏾‍♂️ 

@pmatilai has been documenting things, maybe he's got it written up somewhere?

Reply:
> This should run _after_ all other systemd triggers, since we may need tmpfiles and users populated.

10000 is super low, so it would be *after* most stuff, no?